quiz.js

```diff
@@ -0,0 +1,42 @@
+
+var height =document.getElementById("height");
+var character = document.getElementById("character");
+var treeButton = document.getElementById("keypress-input");
+
+
+function growYourTree(tree) {
+	//console.log("tree", tree );
+		tree = {
+			height: height.value,
+			character: character.value
+		}
+	  
+	  	
+	for (var i = 1; i <= tree.height;  i++ ) {
+		
+	  	var inputChar = tree.character;
+
+		console.log (" ".repeat(height.value--) + inputChar.repeat((i*2)-1));
+ }
+ }
+
+
+	
+
+ 
+function checkInput () {
+	
+	if (height.value && character.value) {
+		//tree.height = height.value;
+		//tree.character = character.value;
+		growYourTree ();
```

Comment:
Not sure why you commented out the assigning of properties to your tree object, because this is a good place to do that. Your `growYourTree` function is defined to accept an argument, but here you're not actually passing an argument to it ( You have an empty parenthesis when you're calling the function )